modin/pandas/test/test_io.py

```diff
@@ -55,20 +55,18 @@
 
 pd.DEFAULT_NPARTITIONS = 4
 
-TEST_PARQUET_FILENAME = "test.parquet"
-TEST_CSV_FILENAME = "test.csv"
-TEST_JSON_FILENAME = "test.json"
-TEST_HTML_FILENAME = "test.html"
-TEST_EXCEL_FILENAME = "test.xlsx"
-TEST_FEATHER_FILENAME = "test.feather"
-TEST_READ_HDF_FILENAME = "test.hdf"
-TEST_WRITE_HDF_FILENAME_MODIN = "test_write_modin.hdf"
-TEST_WRITE_HDF_FILENAME_PANDAS = "test_write_pandas.hdf"
-TEST_STATA_FILENAME = "test.dta"
-TEST_PICKLE_FILENAME = "test.pkl"
-TEST_SAS_FILENAME = os.getcwd() + "/data/test1.sas7bdat"
```

Comment:
Why are you remove this variable and test that using it?

Reply:
This variable refers to non-existent file and test, that uses this variable doesn't create this file (also this test marked as `pytest.skip`).